adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1340,19 +1340,25 @@ func generateRoutePaths(xWso2Basepath, basePath, resourcePath string) string {
 
 // generatePathRegexSegment - generates a regex segment for a given resource path
 // resourcePath - resource path of the api
-// options - boolean parameter to indicate whether to generate path segment for substitution string
-func generatePathRegexSegment(resourcePath string, options ...bool) string {
+// skipEscapeMetaChars - skip escaping meta chars (special chars like `.(){}`.) True this for route substitution path.
+func generatePathRegexSegment(resourcePath string, skipEscapeMetaChars bool) string {
```

Comment:
are we expecting to make `skipEscapeMetaChars` as true somewhere in the code ?

Reply:
Yes, this is set to true when setting the substitution string for the route

Reply:
Got it. I searched the param only in this PR. Seems it was available previously also.